settings.py

```diff
@@ -4,21 +4,21 @@
 LOGGING_FORMAT = "%(asctime)s|%(levelname)s: sdx-rabbit-monitor: %(message)s"
 LOGGING_LEVEL = logging.getLevelName(os.getenv('LOGGING_LEVEL', 'DEBUG'))
 
-PORT = os.getenv('SDX_RABBIT_MONITOR_HOST')
+PORT = os.getenv('SDX_RABBIT_MONITOR_PORT')
 
 RABBITMQ_DEFAULT_PASS = os.getenv('SDX_RABBIT_MONITOR_PASS')
 RABBITMQ_DEFAULT_USER = os.getenv('SDX_RABBIT_MONITOR_USER')
-RABBITMQ_DEFAULT_VHOST = os.getenv('SDX_RABBIT_MONITOR_DEFAULT_VHOST')
+RABBITMQ_DEFAULT_VHOST = '%2f'
 
 RABBIT_URL = 'http://{hostname}:{port}/api/'.format(
     hostname=os.getenv('SDX_RABBIT_MONITOR_RABBIT_HOST'),
     port=os.getenv('SDX_RABBIT_MONITOR_MGT_PORT')
 )
 
-WAIT_TIME = int(os.getenv('SDX_RABBIT_MONITOR_WAIT_TIME'))
+WAIT_TIME = int('120')
```

Comment:
You don't need the int casting

WAIT_TIME = 120

Reply:
Fair enough, it was there when I got there so I assumed there was a reason